searchlib/src/vespa/searchlib/attribute/predicate_attribute.cpp

```diff
@@ -141,11 +141,8 @@ PredicateAttribute::before_inc_generation(generation_t current_gen)
 void
 PredicateAttribute::onSave(IAttributeSaveTarget &saveTarget) {
     LOG(info, "Saving predicate attribute version %d", getVersion());
-    vespalib::string name(getBaseFileName());
```

Comment:
Consider adding the attribute name to the log message above.

Reply:
Log message will be extended in next PR.